holoviews/plotting/plot.py

```diff
@@ -1073,7 +1074,13 @@ def _format_title(self, key, dimensions=True, separator='\n'):
                 dim_title = self._frame_title(key, separator=separator)
             else:
                 dim_title = ''
-            title_format = util.bytes_to_unicode(self.title_format)
+
+            custom_title = (self.title != self.param.params('title').default)
+            if custom_title and self.title_format:
+                self.warning('Both title and title_format set. Using title_format parameter')
+
+            title = self.title_format if self.title_format else self.title
```

Comment:
In other aliasing cases the recommended option takes precedence, I don't feel strongly about it though, so it's up you. The logic would be:

```suggestion
            title = self.title if custom_title or self.title_format is None else self.title_format
```